src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -113,7 +113,7 @@ public function getName()
      */
     public function acceptValue()
     {
-        return $this->isValueRequired() || $this->isValueOptional();
+        return $this->isValueRequired() || $this->isValueOptional() || $this->isArray();
```

Comment:
this change is wrong

Reply:
How passing arrays ?
Like input ``` --option7=[1,2] ``` , output 
``` 
array(2) {
  [0]=>
  int(1)
  [1]=>
  int(2)
}
```